### ISSUE TITLE
Loss is decreasing but MAP as well. What to do ?

### ISSUE THREAD

**Issue by awaisbajwaml** [AlexeyAB/darknet#8028]
CUDA-version: 10000 (10000), cuDNN: 8.4.2, CUDNN_HALF=0, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: Tesla T4
net.optimized_memory = 0

```

With above config and below training results after 22000 iterations, I am getting the below results with two classes. 

Any thoughts?  Why MAP is decreasing? 

![image](https://user-images.githubusercontent.com/47017344/130449578-55a435b9-93e2-40c0-824d-838889c1dee0.png)

**Comment by awaisbajwaml** [comment]
![image](https://user-images.githubusercontent.com/47017344/130449802-ece79abd-ac4f-42f3-a7bb-9a52532be416.png)

**Comment by seungtaek94** [comment]
It seems like occur overfitting.

So u can solve this problem like below:
- Add new data more.
- Do augment data more.
- Reduce learning rate or increase batch size and then stop training earlier before map going down.
- Add dropout layer in ur model.
- Using gradient clipping.
- etc..

**Comment by awaisbajwaml** [comment]
closing this
